wailsapp/wails
# [v2, runtime] Separate runtime js debug build

# Description

Currently, the runtime javascript is only built for `production` and `dev`, so in `debug` the minified version of the runtime is served.
This PR separates the `debug` build of the runtime so it includes the un-minified version with the source map.

# How Has This Been Tested?


## How to test :

In devtools, check `wails/runtime.js` in the `source` tab 

`wails dev` => `wails/runtime.js` is un-minified, `wails/desktop` exists.
`wails build -debug` => `wails/runtime.js` is un-minified, `wails/desktop` exists.
`wails build -devtools` => `wails/runtime.js` is minified, `wails/desktop` doesn't exist.
